roles/cifmw_cephadm/tasks/cephnfs.yml

```diff
@@ -0,0 +1,23 @@
+---
+# Copyright Red Hat, Inc.
+# All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License. You may obtain
+# a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+
+- name: Create NFS Ganesha Cluster
+  ansible.builtin.command: |
+    {{ cifmw_cephadm_ceph_cli }} nfs cluster create {{ cifmw_cephadm_cephfs_name }} \
+    --ingress --virtual-ip={{ cifmw_cephadm_nfs_vip }} \
+    --ingress-mode=haproxy-protocol '--placement={{ placement }}'
```

Comment:
```suggestion
  ansible.builtin.command: >
    {{ cifmw_cephadm_ceph_cli }} nfs cluster create {{ cifmw_cephadm_cephfs_name }}
    --ingress --virtual-ip={{ cifmw_cephadm_nfs_vip }}
    --ingress-mode=haproxy-protocol '--placement={{ placement }}'
```

Reply:
@rebtoor --placement='{{ placement }}' won't work: I already tried that and it should be "--placement={{ placement }}", otherwise cephadm mess up with the command :D

Reply:
+1 to have `>`

Reply:
@fmount yeah, i got it. The problem is in the way how `cifmw_cephadm_ceph_cli` is built. Let's leave it as is now.

Reply:
Done